### PR TITLE
[MSPAINT] Refactoring (_countof and resource.h)

### PR DESCRIPTION
**Pull request by katahiromz**
## Purpose

JIRA issue: [CORE-17931](https://jira.reactos.org/browse/CORE-17931)

## Proposed changes

- Use Microsoft standard `_countof` macro instead of `SIZEOF`.
- Rename `definitions.h` as `resource.h`.
- Move some macro definitions to its proper place.

### CODE REVIEW

File: base/applications/mspaint/common.h [hunk 1]
```diff
@@ -10,6 +10,22 @@
 
 #pragma once
 
+#define GRIP_SIZE 3
+#define MIN_ZOOM 125
+#define MAX_ZOOM 8000
```
Comment (HBelusca) [inline base/applications/mspaint/common.h:15]: With proper indentation/alignment:
```suggestion
#define GRIP_SIZE   3
#define MIN_ZOOM    125
#define MAX_ZOOM    8000
```

File: base/applications/mspaint/precomp.h [hunk 2]
```diff
@@ -21,7 +21,16 @@
...
 
-#include "definitions.h"
+#define WM_TOOLSMODELTOOLCHANGED WM_APP
+#define WM_TOOLSMODELSETTINGSCHANGED (WM_APP + 1)
+#define WM_TOOLSMODELZOOMCHANGED (WM_APP + 2)
+#define WM_PALETTEMODELCOLORCHANGED (WM_APP + 3)
+#define WM_PALETTEMODELPALETTECHANGED (WM_APP + 4)
+#define WM_IMAGEMODELDIMENSIONSCHANGED (WM_APP + 5)
+#define WM_IMAGEMODELIMAGECHANGED (WM_APP + 6)
+#define WM_SELECTIONMODELREFRESHNEEDED (WM_APP + 7)
```
Comment (HBelusca) [inline base/applications/mspaint/precomp.h:31]: Please align/indent the values of these macros. (I know they weren't originally)

File: base/applications/mspaint/resource.h [hunk 3]
```diff
@@ -1,7 +1,7 @@
 /*
  * PROJECT:     PAINT for ReactOS
  * LICENSE:     LGPL
- * FILE:        base/applications/mspaint/definitions.h
+ * FILE:        base/applications/mspaint/resource.h
  * PURPOSE:     Defines the resource ids and other stuff
```
Comment (HBelusca) [inline base/applications/mspaint/resource.h:5]:
```suggestion
 * PURPOSE:     Defines the resource IDs
```

File: base/applications/mspaint/common.h [hunk 4]
```diff
@@ -10,6 +10,22 @@
```
Comment (HBelusca) [inline base/applications/mspaint/common.h:10]: (I cannot comment directly on the line with the github interface)
The "purpose" of this file can be adjusted to mention that there are also commonly used definitions too.

File: base/applications/mspaint/precomp.h (hunk 2 above)
Comment (katahiromz) [reply base/applications/mspaint/precomp.h]: OK.

File: base/applications/mspaint/common.h (hunk 4 above)
Comment (katahiromz) [reply base/applications/mspaint/common.h:10]: OK.

**Review by HBelusca — APPROVED** [review]

**Pull request merged**